proxy/doc.go

```diff
@@ -127,6 +127,19 @@ and `SetOutgoingHost()` methods of the `FilterContext` need to be used
 instead of the `Request.Header` map.
 
 
+Proxy protocol support
```

Comment:
One thing, just suspicious to me. Isn't proxy support added here? https://github.com/bojleros/skipper/blob/1326761471d89c2106d752d80ad9db031542fca9/proxy/doc.go#L155